client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -2015,11 +2023,13 @@
                 });
 
                 metadata.initialize().then(function() {
-                    scope.item.hasCrops = false;
                     scope.metadata = metadata.values;
-                    scope.item.hasCrops = scope.metadata.crop_sizes.some(function (crop) {
-                        return scope.item.renditions[crop.name];
-                    });
+                    if (scope.item.type === 'picture'){
+                        scope.item.hasCrops = false;
```

Comment:
If some piece of code is dependent on `hasCrops` then it might fail. I think it's better to have `scope.itam.hasCrops = false;` outside if.